infra/docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
       - '9999:9999'
     environment:
       GOTRUE_JWT_SECRET: "37c304f8-51aa-419a-a1af-06154e63707a"
-      GOTRUE_JWT_EXP: 3600
+      GOTRUE_JWT_EXP: 90
```

Comment:
was this just for testing? or do you want to keep as 90?

Reply:
oops yeah, just for testing - will revert